### PR TITLE
HACK to make ASFR component work + NeonatalSWC

### PR DESCRIPTION
**Pull request by aflaxman**
### Description

- *Category*: bugfix
- *JIRA issue*: none

### Changes and notes
 These fix a few issues I've run into while messing around lately.

### Testing
small models which didn't work now do

### CODE REVIEW

File: src/vivarium_public_health/population/add_new_birth_cohorts.py [hunk 1]
```diff
@@ -205,8 +205,8 @@ def setup(self, builder: Builder) -> None:
 
     def load_age_specific_fertility_rate_data(self, builder: Builder) -> pd.DataFrame:
         asfr_data = builder.data.load("covariate.age_specific_fertility_rate.estimate")
-        columns = ["year_start", "year_end", "age_start", "age_end", "mean_value"]
-        asfr_data = asfr_data.loc[asfr_data.sex == "Female"][columns]
+        columns = ["year_start", "year_end", "age_start", "age_end", "value"]
+        asfr_data = asfr_data.loc[(asfr_data.sex == "Female") & (asfr_data.parameter == "mean_value")].reset_index()[columns]
```
Comment (albrja) [inline src/vivarium_public_health/population/add_new_birth_cohorts.py:209]: Is it necessary to subset to the second condition here?

**Review by albrja — APPROVED** [review]

File: src/vivarium_public_health/population/add_new_birth_cohorts.py (hunk 1 above)
Comment (aflaxman) [reply src/vivarium_public_health/population/add_new_birth_cohorts.py:209]: I don't remember!

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,3 +1,7 @@
+**3.0.4- 08/20/24**
```
Comment (rmudambi) [inline CHANGELOG.rst:1]: This should be v3.0.3. You should include your change in the block below, as we haven't released 3.0.3 yet

**Review by hussain-jafari — APPROVED** [review]

**Review by rmudambi — APPROVED** [review]
Approved assuming the changelog will be fixed

**Pull request merged**